packages/renderer/src/lib/deployments/DeploymentColumnConditions.svelte

```diff
@@ -3,25 +3,31 @@ import { faCheckCircle, faExclamationTriangle, faQuestionCircle, faSync } from '
 import { Tooltip } from '@podman-desktop/ui-svelte';
 import Fa from 'svelte-fa';
 
-import type { DeploymentUI } from './DeploymentUI';
+import type { DeploymentCondition, DeploymentUI } from './DeploymentUI';
 
 export let object: DeploymentUI;
 
 // Determine both the icon and color based on the deployment condition
-function getConditionAttributes(type: string) {
-  switch (type) {
+function getConditionAttributes(condition: DeploymentCondition) {
```

Comment:
In this function, we are considering that the condition is always "True", but the status of the condition could be "False", which would give another meaning